fix tf.data.Dataset size inferring issue in TF 2.0 keras training loop

angeliand opened the pull request:
fix #33216 issue

angeliand committed:
fix training dataset steps_per_epoch bug

omalleyt12 reviewed (dismissed):
Thanks for the PR! Looks good to me!

rthadur commented on tensorflow/python/keras/engine/training_v2.py:
```diff
@@ -250,13 +250,15 @@ def fit(
           epochs=0)
-
+      
+      steps_per_epoch = inferred_steps if steps_per_epoch is None else steps_per_epoch
```
`FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/keras/engine/training_v2.py:260: [C0301(line-too-long), ] Line too long (86/80)`
can you please check this lint error

angeliand replied:
Hi! Sorry for the late reply. Added some line-breaks according to Google Style Guide, hope it's good now.

angeliand committed:
fixing pylint error

fixing pylint error according to: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#2114-decision
